bokehjs/src/coffee/model.coffee

```diff
@@ -17,6 +19,39 @@ export class Model extends HasProps
       for cb in callbacks
         @listenTo(@, evt, () -> cb.execute(@))
 
+    @listenTo(@, 'change:js_event_callbacks', () -> @_update_event_callbacks)
+
+
+  _process_event : (event) ->
+    # Given an applicable event, execute the associated callback
+    if not event.constructor.event_name
+       console.warn('Event is still passed as a string: ' + event)
```

Comment:
Actually, I expect this branch to be removed once the PR is complete. I'll make sure to use the logger anywhere else I might be using console.